amplify/src/main/java/com/github/stkent/amplify/tracking/checks/CooldownDaysCheck.java

```diff
@@ -33,7 +33,8 @@ public CooldownDaysCheck(final long cooldownPeriodDays) {
 
     @Override
     public boolean shouldAllowFeedbackPrompt(@NonNull final Long cachedEventValue) {
-        return (SystemTimeUtil.currentTimeMillis() - cachedEventValue) >= TimeUnit.DAYS.toMillis(cooldownPeriodDays);
+        return cachedEventValue == Long.MAX_VALUE
+                || (SystemTimeUtil.currentTimeMillis() - cachedEventValue) >= TimeUnit.DAYS.toMillis(cooldownPeriodDays);
```

Comment:
Does this mean we need to change the warmup days check? does that even exist still?